lxd/instance/drivers/driver_qemu.go

```diff
@@ -5073,14 +5026,6 @@ func (d *qemu) Delete(force bool) error {
 				// valid devices to be remove.
 				d.logger.Error("Device remove validation failed", logger.Ctx{"device": entry.Name, "err": err})
 			}
-
-			// If a device was returned from deviceLoad even if validation fails, then try and remove.
```

Comment:
We still need this section.

Reply:
I brought it back here and in `lxd/instance/drivers/driver_lxc.go`